src/Opdex.Platform.Application/EntryHandlers/Tokens/CreateAddTokenCommandHandler.cs

```diff
@@ -53,8 +61,33 @@ public async Task<TokenDto> Handle(CreateAddTokenCommand request, CancellationTo
                           summary.TotalSupply.Value, latestBlock.Height);
 
         var tokenId = await _mediator.Send(new MakeTokenCommand(token, latestBlock.Height));
-        if (tokenId == 0) throw new Exception("Something went wrong when indexing the token.");
+        if (tokenId == 0) _logger.LogError("Something went wrong indexing the token");
```

Comment:
I think throwing here is necessary still or we'll try to insert a wrapped token with tokenId 0

Reply:
True, I've added the exception back.